python/main.py

```diff
@@ -18,14 +21,69 @@
     allow_headers=["*"],
 )
 
+json_file = str(pathlib.Path(__file__).parent.resolve() / "items.json")
+db_file = str(pathlib.Path(os.path.dirname(__file__)
+                           ).parent.resolve() / ".." / "db" / "items.db")
+sqlite_file = str(pathlib.Path(os.path.dirname(__file__)
+                               ).parent.resolve() / ".." / "db" / "mercari.sqlite3")
```

Comment:
I tried to run your code but I need to modify these lines as follows.

```diff
db_file = str(pathlib.Path(os.path.dirname(__file__)
-                           ).parent.resolve() / ".." / "db" / "items.db")
+                           ).resolve() / ".." / "db" / "items.db")

sqlite_file = str(pathlib.Path(os.path.dirname(__file__)
-                               ).parent.resolve() / ".." / "db" / "mercari.sqlite3")
+                               ).resolve() / ".." / "db" / "mercari.sqlite3")
```

Does it work without the above in your environment?
If it works, you don't need to care at the moment.

(My environment is macOS and Python 3.10.2

Reply:
It works on my end without removing `parent` and doesn't otherwise (It shows `Internal Server Error` and on the application log it says, `sqlite3.OperationalError: database is locked`.)

My environment is Windows and Python 3.8.3.